libcxx/test/support/MinSequenceContainer.h

```diff
@@ -47,6 +54,13 @@ struct MinSequenceContainer {
     return from_vector_iterator(data_.insert(to_vector_iterator(p), std::move(value)));
   }
 
+#if defined(__cpp_lib_containers_ranges) && __cpp_lib_containers_ranges >= 202202L
```

Comment:
Is there a reason to provide this conditionally?

Reply:
I don't know. If there's no pre-C++23 mode for this class, it can be dropped.

Reply:
I'm not sure whether we want to use `MinSequenceContainer` with C++98 container adaptors. If so, perhaps it's better to keep the guard.